internal/worker/export_client_test.go

```diff
@@ -0,0 +1,8 @@
+// helper functions for client_test.go
+package worker
+
+func (c *Client) InvalidateAccessToken() {
```

Comment:
I like testing this scenario, but maybe this is for another PR? Doesn't feel related to the crashing.

Reply:
This is mainly because the tests did cover this before, so I wanted to restore/keep parity here

Reply:
Should I mention that the test behavior changed and split to another PR? no big deal, if needed